### ISSUE TITLE
Uploaded images naming problem in 2.2.3

### ISSUE THREAD

**Issue by freke1981** [n2cms/n2cms#711]
We use file upload fields in some parts to upload images. Recently the naming of the uploaded files have startade to change, without any cause. Like XXXXXXX_smallspot.jpg the smallspot is one of the sizes the image is in. On some images the the word smallspot is added several times like  XXXXXXX_smallspot_smallspot_smallspot_smallspot.jpg. I also seem to affect only some images not all on the site, I´ve checked the database and wrong names are already there. What in N2 can cause that several images changes their names?

**Comment by libardo** [comment]
Are you storing files in the database? I tried reproducing but no luck. Could you provide the exact steps to reproduce this on one of the templates projects? Thans!

**Comment by freke1981** [comment]
Yes, we are storing the images in the database. We use an "image/file
upload" field in the page type controller. And then we just get the file
url and append "_smallspot" before the file extension before sending the
filename to the view. It has worked for several months, but one day the
"_smallspot" already were on the file name when we got it from the db. So
after we appended it we got a filename like XXXX_smallspot_smallspot.jpg.
And that is not the correct filename.
